operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -400,20 +418,31 @@ Please refer [iSCSI Metrics](https://github.com/Cray-HPE/sbps-marshal/blob/main/
 
 ## Steps to switch from DVS to iSCSI
 
+This involves below sequence of operations:
+
 * Disable DVS
 * Uninstall CPS
 * Enable iSCSI SBPS
 
-To disable DVS please refer:
-<https://github.hpe.com/hpe/hpc-shasta-os-uss-docs/blob/master/dac-assets/documents/operations/DVS_and_SBPS.md>
+To perform above operations, please refer the section `Switch from DVS to iSCSI` documented in the publication:
+`HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray Supercomputing EX Systems (S-8063)`.
+
+## Steps to continue using DVS based projection
+
+If user wants to continue using DVS then during the BOS session template creation (manually using `cray bos` command)
+below parameter values have to be used:
 
-To uninstall CPS services please refer:
-<https://github.hpe.com/hpe/hpc-shasta-os-uss-docs/blob/master/dac-assets/documents/operations/Uninstall_CPS.md>
+```bash
```

Comment:
This isn't a bash script, so remove `bash`

Reply:
There was an error for this to have language, let me try this time.

Reply:
Ugh, the lint code is forcing the "optional" language identifier.
Try ` ```text` or ` ```plain text`

Reply:
ok.

Reply:
This went fine now. Thanks David!